app/models/authenticator/httpd.rb

```diff
@@ -48,5 +52,45 @@ def update_user_attributes(user, _username, identity)
       user.last_name  = user_attrs[:lastname]
       user.email      = user_attrs[:email] unless user_attrs[:email].blank?
     end
+
+    def get_user_details_by_userid(username)
```

Comment:
If the `get_` is removed from `get_user_details_by_userid` is should also be removed from: 
`get_user_details_from_headers`